File: src/main/java/phoenix/items/ash/CrucibleItem.java
package phoenix.items.ash;

import net.minecraft.item.Item;
import net.minecraft.item.ItemStack;
import phoenix.Phoenix;
import phoenix.init.PhoenixItems;

public class CrucibleItem extends Item
{
    public CrucibleItem()
    {
        super(new Item.Properties().group(Phoenix.PHOENIX).maxStackSize(1));
    }

    @Override
    public boolean hasContainerItem()
    {
        return true;
    }

    @Override
    public ItemStack getContainerItem(ItemStack stack)
    {
        return new ItemStack(PhoenixItems.CRUCIBLE.get());
    }
}

File: src/main/kotlin/phoenix/items/ash/CrucibleItem.kt
package phoenix.items.ash

import net.minecraft.item.Item
import net.minecraft.item.ItemStack
import phoenix.Phoenix
import phoenix.init.PhxItems

class CrucibleItem : Item(Properties().group(Phoenix.ASH).maxStackSize(1).containerItem(PhxItems.CRUCIBLE))
{
    override fun hasContainerItem(): Boolean = true

    override fun getContainerItem(stack: ItemStack): ItemStack = ItemStack(PhxItems.CRUCIBLE)
}